Sming/Services/SpifFS/spiffs_sming.c

```diff
@@ -118,64 +120,62 @@ static void spiffs_mount_internal(spiffs_config *cfg)
-			  debugf("SPIFFS_USE_MAGIC is set!\r\n");
-			  if(res == SPIFFS_ERR_NOT_A_FS) {
-				  debugf("Error SPIFFS_ERR_NOT_A_FS which is expected, continuing with format...\r\n");
-				  spiffs_unmount();
-				  spiffs_format_internal(cfg);
-			  }
-		  } else {
-			  debugf("Formatting fs...\r\n");
+		  debug_d("Unsuccessful SPIFFS check with error %d.\r\n", check);
+		  debug_w("ALL DATA WILL BE ERASED. First init filesystem.\r\n");
+
+#if(SPIFFS_USE_MAGIC == 1)
+		  // See https://github.com/pellepl/spiffs/wiki/Using-spiffs - Formatting
+		  debug_d("SPIFFS_USE_MAGIC is set!\r\n");
+		  if(res == SPIFFS_ERR_NOT_A_FS) {
+			  debug_d("Error SPIFFS_ERR_NOT_A_FS which is expected, continuing with format...\r\n");
 			  spiffs_unmount();
 			  spiffs_format_internal(cfg);
 		  }
+#else
+		  debug_i("Formatting fs...\r\n");
+		  spiffs_unmount();
+		  spiffs_format_internal(cfg);
+#endif
 		  res = spiffs_mount_minimal(cfg);
-		  debugf("Mount result is: %d\r\n", res);
+		  debug_i("Mount result is: %d\r\n", res);
 	  } else {
-		  debugf("SPIFFS check was successful (return code %d)\r\n", check);
-		  debugf("Trying to mount FS\r\n");
+		  debug_d("SPIFFS check was successful (return code %d)\r\n", check);
 		  res = spiffs_mount_minimal(cfg);
 		  if(res < 0) {
-			  debugf("UNSUCCESSFUL FS mount after SPIFFS check! Continuing anyway! PAY ATTENTION!\r\n");
+			  debug_w("UNSUCCESSFUL FS mount after SPIFFS check! Continuing anyway! PAY ATTENTION!\r\n");
 		  }
 	  }
   }
 
   file_t fd = SPIFFS_open(&_filesystemStorageHandle, "initialize_fs_header.dat", SPIFFS_RDONLY, 0);
   SPIFFS_close(&_filesystemStorageHandle, fd);
   if(fd < 0) {
-	  debugf("No initialized SPIFFS volume found. Wiping flash and initializing FS now...");
+	  debug_i("No initialized SPIFFS volume found. Wiping flash and initializing FS now...");
 	  spiffs_unmount();
 	  spiffs_format_internal(cfg);
 	  res = spiffs_mount_minimal(cfg);
 	  if(res < 0) {
-		  debugf("\r\nCouldn't mount SPIFFS even on freshly wiped flash. Giving up. :(\r\n");
+		  debug_w("\r\nCouldn't mount SPIFFS even on freshly wiped flash. Giving up. :(\r\n");
 	  } else {
-		  fd = SPIFFS_open(&_filesystemStorageHandle, "initialize_fs_header.dat",
-						   SPIFFS_CREAT | SPIFFS_TRUNC | SPIFFS_RDWR, 0);
+		  fd = SPIFFS_open(&_filesystemStorageHandle, "initialize_fs_header.dat", SPIFFS_CREAT | SPIFFS_TRUNC | SPIFFS_RDWR, 0);
 		  SPIFFS_write(&_filesystemStorageHandle, fd, (u8_t*)"1", 1);
 		  //SPIFFS_fremove(&_filesystemStorageHandle, fd);
 		  SPIFFS_close(&_filesystemStorageHandle, fd);
 		  fd = SPIFFS_open(&_filesystemStorageHandle, "initialize_fs_header.dat", SPIFFS_RDONLY, 0);
 		  SPIFFS_close(&_filesystemStorageHandle, fd);
 		  if(fd < 0) {
-			  debugf("Flash was wiped and mount successfully, but SPIFFS initialization failed. Giving up.");
+			  debug_w("Flash was wiped and mount successfully, but SPIFFS initialization failed. Giving up.");
```

Comment:
"Flash was wiped and mount**ed** successfully, but SPIFFS initialization failed. Giving up."